doc/man1/openssl.pod

```diff
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-openssl - OpenSSL command line tool
+openssl - OpenSSL command line program
```

Comment:
Unsure here. Tool looks better to me but I do not insist.

Reply:
left this one.